### PR TITLE
Fix a few compiler issues on Windows, but not all

### PR DESCRIPTION
**Pull request by sgpearse**
This fixes some of the windows build errors, but we are still failing on the render, osgl_test_version, osgl_test_framebuffer_glad, vapi, vaporpychecker, vapor, and Package solutions.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -130,6 +130,8 @@ else()
 endif()
 
 include_directories ("${PROJECT_SOURCE_DIR}/include")
+include_directories ("${PROJECT_SOURCE_DIR}/lib/osgl/include")
+include_directories ("${PROJECT_SOURCE_DIR}/lib/osgl/glad/include")
```
Comment (StasJ) [inline CMakeLists.txt:134]: These are already included in osgl/CMakeLists.txt

File: CMakeLists.txt [hunk 2]
```diff
@@ -159,18 +161,19 @@ function(FIND_BUNDLED_PYTHON)
...
     set(Python_VERSION "${PYTHONVERSION}")
-    set(Python_SITELIB "${PYTHONPATH}/site-packages")
     set(Python_NumPy_INCLUDE_DIRS "${NUMPY_INCLUDE_DIR}")
     unset(Python_LIBRARIES) # This is required for find_library to work in certain cases
     find_library(
         Python_LIBRARIES
-        NAMES python${PYTHONVERSION} python${PYTHONVERSION}m
-        PATHS "${THIRD_PARTY_LIB_DIR}"
+	NAMES python${PYTHONVERSION} python${PYTHONVERSION}m
+	PATHS ${THIRD_PARTY_LIB_DIR} ${PYTHONPATH}
```
Comment (StasJ) [inline CMakeLists.txt:169]: Omitting quotes may break the build.

File: lib/osgl/CMakeLists.txt [hunk 3]
```diff
@@ -112,6 +112,8 @@ foreach (TEST ${TESTS})
     set(TEST_TARGET "${LIB}_${TEST_NAME}")
     add_executable(${TEST_TARGET} "${TEST}")
     target_link_libraries(${TEST_TARGET} ${LIB})
+if (NOT WIN32)
     target_compile_options(${TEST_TARGET} PRIVATE -Wno-deprecated-declarations)
+endif()
```
Comment (StasJ) [inline lib/osgl/CMakeLists.txt:117]: Indentation

**Comment by sgpearse** [comment]
Closing with the merge of #3185.

**Pull request closed without merging**